charts/substra-backend/values.yaml

```diff
@@ -376,6 +376,27 @@ worker:
         memory: "1Gi"
       limits:
         memory: "64Gi"
+    ## @param worker.computePod.nodeSelector Node labels for pod assignment
+    ##
+    nodeSelector: {}
+    ## @param worker.computePod.tolerations Toleration labels for pod assignment
+    ##
+    tolerations: []
+    ## @param worker.computePod.affinity.podAffinity.requiredDuringSchedulingIgnoredDuringExecution[0].labelSelector.matchExpressions[0].key Pod affinity rule defnition.
+    ## @param worker.computePod.affinity.podAffinity.requiredDuringSchedulingIgnoredDuringExecution[0].labelSelector.matchExpressions[0].operator Pod affinity rule defnition.
+    ## @param worker.computePod.affinity.podAffinity.requiredDuringSchedulingIgnoredDuringExecution[0].labelSelector.matchExpressions[0].values Pod affinity rule defnition.
+    ## @param worker.computePod.affinity.podAffinity.requiredDuringSchedulingIgnoredDuringExecution[0].topologyKey Pod affinity rule defnition.
+    ##
+    affinity:
+      podAffinity:
+        requiredDuringSchedulingIgnoredDuringExecution:
+        - labelSelector:
+            matchExpressions:
+            - key: statefulset.kubernetes.io/pod-name
+              operator: In
+              values:
+              - $(POD_NAME)
```

Comment:
Is it planned to add a page in the documentation about how to configure these values to be able to use a GPU node spawned on demand?
Or do you think it's too specific to the cloud provider?

Reply:
I don't think it is planned, and I would argue that even if decide to plan it, it is not in this PR, which does not have anything specificly related to GPU.

(Would like feedback from @RomainGoussault  and @thbcmlowk on this topic)

Reply:
It is is something we want to document